### DISCUSSION TITLE
Use for ASL

### DISCUSSION THREAD

**Discussion by mvpapen** [sircharlo/meeting-media-manager#330] [Ideas] [answered]
1.could this program be adapted for ASL use ?  (Integration of orange app)
2. and  maybe integration of  2Ptz cameras (KH camera control) by counter agent?

**Comment by sircharlo** [comment, marked as answer]
Regarding ASL or sign language use: I just took a look at a meeting program in ASL.
I understand that the material itself is presented as videos, to allow for all to read and prepare it.
What parts of the actual workbook are actually used/played during meetings though? I'm just wondering if all media files would need to be included, including all source material, or if there's a specific way that it's usually done. 
As for the Watchtower, I suppose the whole file would need to be downloaded, and I would need to also add support for "markers", which is how different paragraphs are identified for easy navigation.
Any insights as to what would be needed (or not needed) could be helpful, as I have never been to a sign language meeting unfortunately.

For the camera, you mean you would like to be able to control a second camera from M³ (presumably your audience camera)? Do you use OBS Studio?
